deployment/helm/ditto/values.yaml

```diff
@@ -131,6 +131,19 @@ global:
     path: "/"
     # port where prometheus metrics will be provided
     port: 9095
+  # featureFlags contains feature flags enabling/disabling certain features
+  featureFlags:
+    # mergeThingsEnabled controls whether MergeThing commands (patching of things) are enabled or not
+    mergeThingsEnabled: true
+    # wotIntegrationEnabled controls whether the Web of Things (WoT) integration is enabled or not
+    wotIntegrationEnabled: true
+    # historicalApisEnabled controls whether the historical APIs (using the event journal) are enabled or not
+    historicalApisEnabled: true
+    # preserveKnownMqttHeadersEnabled controls whether known MQTT headers (e.g., mqtt.topic) are preserved in outgoing messages
+    preserveKnownMqttHeadersEnabled: true
+    # jsonKeyValidationEnabled controls whether for each json object, each JSON key should be validated against a regex pattern
+    #  excluding control characters to be used as JSON keys. This can have quite a performance impact.
+    jsonKeyValidationEnabled: true
```

Comment:
shouldn't be added to the things-deployment.yaml as well since jsonKeyValidation is used in ThingsModelFactory?

Reply:
@kalinkostashki good catch .. that was not on purpose .. I seem to have forgotten it :/